go.mod

```diff
@@ -8,22 +8,28 @@ go 1.12
 
 require (
 	bitbucket.org/bertimus9/systemstat v0.0.0-20180207000608-0eeff89b0690
-	github.com/Azure/azure-sdk-for-go v21.4.0+incompatible
```

Comment:
Why the go.mod changes?

Reply:
When I did the rebase, there were some changes to that file. I could have rebased some of the wrong changes? :thinking: